docs/examples/te_llama/te_llama.py

```diff
@@ -121,12 +121,10 @@ def from_pretrained_local(cls, pretrained_model_name_or_path, *args, config, **k
             assert not isinstance(resolved_archive_file, list)
             resolved_archive_file = [resolved_archive_file]
 
-        error_msgs = []
         for shard_file in resolved_archive_file:
             state_dict = load_state_dict(shard_file)
-            replaced_layers = replace_params(state_dict, vanilla_model.state_dict())
-
-            error_msgs += _load_state_dict_into_model(vanilla_model, state_dict, start_prefix="")
+            replace_params(state_dict, vanilla_model.state_dict())
+            _load_state_dict_into_model(vanilla_model, state_dict, start_prefix="")
```

Comment:
Now I feel that this line is maybe not needed. Could you verify, @pggPL ?

Reply:
Seems that it's necessary - results without it are different than with it. I will investigate it tomorrow.

Reply:
We do not load all params in replace_params - for example embedding, thus this line is necessary.